src/libs/models/BankAccount.ts

```diff
@@ -0,0 +1,241 @@
+import Str from 'expensify-common/lib/str';
+import CONST from '../../CONST';
+import { ValueOf } from "type-fest";
+import BankAccountJSON, {AdditionalData} from "../../types/onyx/BankAccount";
+
+type State = ValueOf<typeof BankAccount.STATE>;
+
+type BankAccountType = "withdrawal" | "direct-deposit";
+
+type ACHData = {
+    routingNumber: string,
+    accountNumber: string,
+    addressName: string,
+    isSavings: boolean,
+    bankAccountID: number,
+    state: State,
+    validateCodeExpectedDate: string,
+    needsToUpgrade: boolean,
+};
+
+class BankAccount {
+    public static readonly STATE = {
```

Comment:
It is public by default.
```suggestion
    static readonly STATE = {
```